src/App.js

```diff
@@ -1,11 +1,26 @@
 import "./App.css";
+import Homepage from "./Components/Homepage"
+import Navbar from "./Components/Navbar"
+import Movies from "./Components/Movies"
+import People from "./Components/People"
+import Locations from "./Components/Locations"
+import ErrorBoundary from "./ErrorBoundary"
+import { Route, Switch } from "react-router-dom";
+
 
 function App() {
   return (
     <div className="app">
-      <main>
-        <h1>Hello, world!</h1>
-      </main>
+      <ErrorBoundary>
+        <Navbar />
```

Comment:
I probably wouldn't put the NavBar inside the ErrorBoundary. If something breaks, you want your users to be able to still navigate away from the page.